Xamarin.Forms.Platform.GTK/Renderers/ScrollViewRenderer.cs

```diff
@@ -196,5 +202,27 @@ private void UpdateContentSize()
 
             Control.SetSizeRequest(width, height);
         }
+
+        void UpdateVerticalScrollBarVisibility()
+        {
+            Control.VscrollbarPolicy = ScrollBarVisibilityToGtk(Element.VerticalScrollBarVisibility);
+        }
+
+        void UpdateHorizontalScrollBarVisibility()
+        {
+            if (Element.Orientation == ScrollOrientation.Horizontal || Element.Orientation == ScrollOrientation.Both)
```

Comment:
`var orientation = Element.Orientation;` then use `orientation` wherever you need it. This reduces the number of calls to BindableObject.GetValue which is effectively a list lookup.